src/main/java/subway/domain/LineRepository.java

```diff
@@ -4,19 +4,61 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.Objects;
+import java.util.stream.Collectors;
+import subway.AppStatusCode;
 
 public class LineRepository {
-    private static final List<Line> lines = new ArrayList<>();
 
-    public static List<Line> lines() {
+    private final static List<Line> lines = new ArrayList<>();
+
+    public List<Line> getLines() {
         return Collections.unmodifiableList(lines);
     }
 
-    public static void addLine(Line line) {
+    private List<Line> searchLine(String lineName) {
+        return lines.stream().filter(line -> line.getName().equals(lineName))
+            .collect(Collectors.toList());
```

Comment:
```suggestion
        return lines.stream()
            .filter(line -> line.getName().equals(lineName))
            .collect(Collectors.toList());
```
체이닝을 사용할때는 .마다 내리면 가독성이 증가합니다.

Reply:
체이닝 시에 .마다 내리는 점에 대해서 이번 기회에 알게 되었습니다. 향후 피드백 반영에 참조하겠습니다.